tests/common.rs

```diff
@@ -0,0 +1,44 @@
+use axum::Router;
+use blockfrost_platform::{
+    cli::{Config, LogLevel, Mode, Network},
+    server::build,
+    AppError, NodePool,
+};
+use std::{
+    env::var,
+    sync::{Arc, LazyLock},
+};
+use tower_http::normalize_path::NormalizePath;
+
+static INIT_LOGGING: LazyLock<()> = LazyLock::new(|| {
+    tracing_subscriber::fmt::init();
+});
+
+pub fn initialize_logging() {
+    let _ = INIT_LOGGING;
+}
+
+pub fn test_config() -> Arc<Config> {
+    let node_socket_path_env =
+        var("NODE_SOCKET_PATH").unwrap_or_else(|_| "/run/cardano-node/node.socket".into());
```

Comment:
Maybe `use std::env` + more qualified `env::var("NODE_SOCKET_PATH")`, since `var` is a pretty generic name, and `env::var` is immediately understandable. But whichever you prefer. :)